tests/resources/promise-rejection-events.js

```diff
@@ -446,14 +446,14 @@ async_test(function(t) {
    'all inside a setTimeout');
 
 //
-// Positive unhandledrejection/rejectionhandled tests with delayed attachment
+// Negative unhandledrejection/rejectionhandled tests with task-delayed attachment
 //
 
 async_test(function(t) {
   var e = new Error();
   var p;
 
-  onUnhandledSucceed(t, e, function() { return p; });
+  onUnhandledFail(t, function() { return p; });
```

Comment:
Why? I thought that was the entire content of the discussion in #2. We should not send events if the promise has been handled by the time we *would* send the events.